internal/common/function/function.go

```diff
@@ -34,44 +34,21 @@ import (
 	"github.com/migtools/oadp-non-admin/internal/common/constant"
 )
 
-const requiredAnnotationError = "backup does not have the required annotation '%s'"
-
-// AddNonAdminLabels return a map with both the object labels and with the default Non Admin labels.
-// If error occurs, a map with only the default Non Admin labels is returned
-func AddNonAdminLabels(labels map[string]string) map[string]string {
-	defaultLabels := map[string]string{
+// AddNonAdminLabels return the required Non Admin labels
+func AddNonAdminLabels() map[string]string {
+	return map[string]string{
 		constant.OadpLabel:      constant.OadpLabelValue,
 		constant.ManagedByLabel: constant.ManagedByLabelValue,
 	}
-
-	mergedLabels, err := mergeMaps(defaultLabels, labels)
-	if err != nil {
-		// TODO logger
-		_, _ = fmt.Println("Error merging labels:", err)
-		// TODO break?
-		return defaultLabels
-	}
-	return mergedLabels
 }
 
-// AddNonAdminBackupAnnotations return a map with both the object annotations and with the default NonAdminBackup annotations.
-// If error occurs, a map with only the default NonAdminBackup annotations is returned
-func AddNonAdminBackupAnnotations(ownerNamespace string, ownerName string, ownerUUID string, existingAnnotations map[string]string) map[string]string {
```

Comment:
@mateusoliveira43 why to remove `mergeMaps` from this function? Similarly from AddNonAdminLabels? The idea was to be able to add labels/annotations to whatever object had in a first place and propagate this to the Velero object.

Current function is completely modified, so if the user creates objects containing already existing annotations or labels which is perfectly valid scenario, those will get overridden/lost.

Reply:
The function name was `AddNonAdminBackupAnnotations` so Add NonAdminBackup annotations to the object annotations.

You are just modyfing this function to:

`GetNonAdminBackupDefaultAnnotations`

Reply:
Taking `AddNonAdminBackupAnnotations` as example

It is only used here in the code, right?
https://github.com/migtools/oadp-non-admin/blob/f99b25cc3b05ab28993294947bcf5d482bc7ee5c/internal/controller/nonadminbackup_controller.go#L258

In that place, the controller is creating the Velero Backup, so, there will never be a annotation conflict, right?

Same idea for labels.

So, I removed the check from function because they were unnecessary

Reply:
So in this case I think we don't need that function at all and we can simply specify those labels around the above code, this way we won't expose confusing public function (confising, because we don't have now confusing name that it tells it's adding labels/annotations) ?

Reply:
or rename those functions to something more meaningful - current function of it.

Reply:
yes, let's rename then

what is your suggestion?

Reply:
`GetNonAdminBackupAnnotations`